### ISSUE TITLE
[Suggestion] HT`s Tree Chop

### ISSUE THREAD

**Issue by Crewm8** [TeamMoegMC/TheWinterRescue#68]
### Pain Point

The current mod for tree chopping is... Suboptimal in my opinion. Yes it does a good job cutting trees pretty fast, but the sheer amount of damage to the tool after cutting a big tree.

### Suggestion

I would like to suggest HT`s Tree Chop mod instead of one that you are using tight now. It takes less durability of the tool while consuming more time to chop the tree. 

### Modpack version

[Beta] 0.5.5 stable 1 -hf1

**Comment by SecretSeptus** [comment]
Have you tried Tetra's axes? they are much more durable than default bronze axes.

**Comment by Crewm8** [comment]
Yes, took me a while to understand that I can make them (I wasted 6 iron axes *immeasurable pain*), even tho they`re durable the amounts of required wood is still obliterates them rather quikckly. I needed 2 of them to get the required amount of wood in one go. I`d still suggest using TreeChop since it uses smaller amount of durability while taking more time to take down one tree (especially if it is a big one).

**Comment by MisterSchmutz** [comment]
Maybe #62 would be solvable with this too? Kinda breaks early chopping progression sadly :( 

Actually how is the tree chopping solved right now, cant seem to find a mod for it in modlist?

**Comment by yuesha-yc** [comment]
We added this in 1.20 branch.
